IdentityCore/src/validation/MSIDAuthority.h

```diff
@@ -92,6 +92,9 @@ typedef void(^MSIDOpenIdConfigurationInfoBlock)(MSIDOpenIdProviderMetadata * _Nu
 // Only certain authorities support MAM CA scenarios
 - (BOOL)supportsMAMScenarios;
 
+// Check if token endpoint is the same cloud as the authoirty
+- (BOOL)checkTokenEndpointForRTRefresh:(nullable NSURL *)tokenEndpoint;
```

Comment:
Should we mentioned that we should resolve authority prior to calling this method?

Reply:
Added :)